tests/fixtures/ci/docker/alpine.Dockerfile

```
@@ -8,34 +8,38 @@
-RUN curl -fsSL https://go.dev/dl/go1.23.4.linux-amd64.tar.gz | tar -C /usr/local -xzf - \
-    && ln -sf /usr/local/go/bin/go /usr/local/bin/go
-ENV PATH="/usr/local/go/bin:/root/.local/bin:/root/.cargo/bin:/root/.local/share/mise/shims:${PATH}"
-# End SECTION: managed tool bootstrap
-
 WORKDIR /workspace
+# init-setup.sh is the single declarative bootstrap: it downloads one throwaway
+# mise, projects the managed registry from the typed SSOT (config/tools.yaml),
+# installs everything mise manages (mise, uv, go, node, gh, docker...) including
+# itself, then hands control to the managed copy and deletes the throwaway.
+# The GITHUB_TOKEN build-arg authenticates GitHub API reads during provisioning.
+ARG GITHUB_TOKEN
```

Hadolint (2.15.1)

[warning] Potentially sensitive data should not be used in the `ARG` or `ENV` commands

(DL3064)